lib/content-services/src/lib/search/services/search-facet-filters.service.ts

```diff
@@ -335,6 +342,25 @@ export class SearchFacetFiltersService implements OnDestroy {
         };
     }
 
+    private loadCategoryNames(bucketList: FacetFieldBucket[]) {
+        bucketList.forEach((item) => {
+            const categoryId = item.label.split('/').pop();
+            this.categoryService.getCategory(categoryId)
+                .subscribe(categoryEntry => {
+                    this.categoryService.searchCategories(categoryEntry.entry.name).subscribe(
```

Comment:
In case you have two nested subscribes when nested depends on top subscribe result you may replace two separated subscribe with single one thanks to concatMap: 

https://www.learnrxjs.io/learn-rxjs/operators/transformation/concatmap

Reply:
good point, thanks